holoviews/plotting/bokeh/util.py

```diff
@@ -177,3 +179,33 @@ def hsv_to_rgb(hsv):
     rgb = clist[order[i], np.arange(np.prod(shape))[:,None]]
 
     return rgb.reshape(shape+(3,))
+
+
+def update_plot(old, new):
```

Comment:
Agreed, they're working on it. Will make a note that it should be replaced after bokeh 0.12.